docs/overview/Glossary.md

```diff
@@ -294,6 +295,10 @@ A _submitted ballot_ is an encrypted ballot that has been [cast](#casting-a-ball
     a1 --> |spoiled| a2
 ```
 
+### tally
+
+The grouping of all selections made for each option on every contest in the election.
```

Comment:
This term seemed important.  Just made up the description from bits I've been gathering, please double-check.